webview/src/experiments/components/table/styles.module.scss

```diff
@@ -336,6 +378,21 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
           background-color: $row-bg-selected-color;
         }
       }
+
+      .indicatorCount {
+        background-color: $indicator-selected-badge-background;
+        color: $indicator-selected-badge-foreground;
+        border: 1px solid $indicator-selected-badge-border;
+      }
+
+      .starSwitch:not([aria-checked='true']) svg {
+        fill: $star-switch-selection-fill-color;
+      }
+
+      .expandedRowArrow,
```

Comment:
Went ahead and updated the cursor color when selected as well since it's initially the same color as the stars:

<img width="161" alt="image" src="https://user-images.githubusercontent.com/43496356/189001268-881b121b-ca6b-4414-8949-1a31dd1b3cde.png">

Reply:
Feels like we are overcomplicating this. Chevrons and badges do not need to change color:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/189031802-fdadd9a2-6c59-46e8-9ba5-e0679145f3ea.png">

Reply:
> Feels like we are overcomplicating this. Chevrons and badges do not need to change color:

By badges, I'm assuming you mean the indicator counts? Agreed, that these would probably be ok to leave as is! cc @maxagin. 

As for chevrons, updating the color does improve contrast on a couple different tested themes, including [VSCode's default light theme](https://github.com/iterative/vscode-dvc/pull/2351#discussion_r965383603) 🤔